v1/options.go

```diff
@@ -205,12 +205,12 @@ func ReportLegacyErrorValues(v bool) Options {
 // such a kind. Specifically, `string` will not stringify bool, string,
 // or numeric kinds within a composite data type
 // (e.g., array, slice, struct, map, or interface).
+// A JSON null quoted in a JSON string is a valid substitute for JSON null
+// while unmarshaling into fields that `string` takes effect on.
```

Comment:
nit: "while unmarshaling" sort of contradicts "this affects either marshaling or unmarshaling" below. can we rewrite this to be a bit less confusing? at first I thought I was blind because I couldn't see where you marshalled stringified nulls.